geogem.rb

```diff
@@ -45,9 +45,20 @@ def self_intersects?
   end
 
   def intersects?(poly)
-    return true unless !bbox_intersect?(self.bbox, poly.bbox)
-    return true unless !poly.nodes.each { |n| self.point_in_poly?(n) }.any?
-    self.nodes.each { |n| poly.point_in_poly?(n) }.any?
+    #binding.pry
```

Comment:
Don't be lazy, take out the debuggers ;)

Reply:
oops... Sorry about that.